tech-debt.md

```diff
@@ -2,5 +2,88 @@
 
 > a concept in programming that reflects the extra development work that arises when code that is easy to implement in the short run is used instead of applying the best overall solution
 
-We don't currently have a strategy for technical debt, at least not documented. This document aims to do that.
+Tech Debt is an inevitable part of programming, a valuable tool even. Do not shy away from it. Embrace it! Subdue it! Make it your servant. What ever you do, *do not just ignore it* and hope it goes away.
```

Comment:
FYI you can edit typos like this yourself

* View full changes
* Pencil top left
* Make change
* Commit directly

Quicker that commenting and having the creator correct